### PR TITLE
Adds translations for spanish, portuguese and french and placeholder for indonesian

### PR DESCRIPTION
**Pull request by simaob**

### CODE REVIEW

File: app/locales/fr.json [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
+    "confirm": "Confirmer"
 	},
 	"dashboard": {
-		"title": "Dashboard",
-		"alerts": "See my alerts",
-		"reports": "Make a report",
-		"myReports": "My reports",
-		"uploadData": "Saved and uploaded data",
-		"map": "Map"
+		"title": "Panel",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Tableau de bord

File: app/locales/es.json [hunk 2]
```diff
@@ -1,87 +1,87 @@
...
+		"next": "Siguiente",
+		"back": "Atrás",
+		"finish": "Terminar",
+		"setUp": "Configuración",
+    "cancel": "Cancelar",
+    "confirm": "Confirmar"
 	},
 	"dashboard": {
-		"title": "Dashboard",
+		"title": "Cuadro de mandos",
```
Comment (davidsingal) [inline app/locales/es.json]: Panel de control

File: app/locales/fr.json [hunk 3]
```diff
@@ -1,87 +1,87 @@
...
-		"drawArea": "Draw an area"
+		"chooseBoundaries": "Maintenant, choisissez les limites",
+		"selectProtectedArea": "Sélectionnez une zone protégée",
+		"drawArea": "Dessinez une zone"
 	},
 	"setupCountry": {
-		"welcomeText": "Hi",
-		"secondWelcomeText": "please set up an area",
-		"firstInstruction": "First, select your country of interest"
+		"welcomeText": "Salut",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Bonjour

"Salut" is quite informal. Depends on the tone you want to give but I would suggest using "Bonjour".

File: app/locales/fr.json [hunk 4]
```diff
@@ -1,87 +1,87 @@
...
+		"secondWelcomeText": "veuillez créer une zone",
+		"firstInstruction": "Tout d'abord, sélectionnez votre pays d'intérêt"
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Not sure about this one but I don't know what could be better.

File: app/locales/fr.json [hunk 5]
```diff
@@ -1,87 +1,87 @@
...
 	},
 	"setupDrawAreas": {
-		"tapInstruction": "Tap on map to draw"
+		"tapInstruction": "Appuyez sur la carte pour dessiner"
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Touchez la carte pour dessiner

File: app/locales/es.json [hunk 6]
```diff
@@ -1,87 +1,87 @@
...
+		"reports": "Hacer un informe",
+		"myReports": "Mis informes",
+		"uploadData": "Datos guardados y cargados",
+		"map": "Mapa"
 	},
 	"login": {
-		"title": "Login",
-		"introductionText": "Sign in with a MyGFW account",
+		"title": "Iniciar sesión",
+		"introductionText": "Iniciar sesión con una cuenta MyGFW",
```
Comment (davidsingal) [inline app/locales/es.json]: Iniciar sesión con una cuenta de MyGFW

File: app/locales/fr.json [hunk 7]
```diff
@@ -1,87 +1,87 @@
...
-		"next": "Next",
-		"back": "Back",
-		"finish": "Finish",
-		"setUp": "Set up",
-    "cancel": "Cancel",
-    "confirm": "Confirm"
+		"next": "Suivant",
+		"back": "Retourner",
+		"finish": "Terminer",
+		"setUp": "Installer",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: If it's a one time thing (like set up an account): Configurer

File: app/locales/es.json [hunk 8]
```diff
@@ -1,87 +1,87 @@
...
-		"errorFirst": "Invalid prop",
-		"errorSecond": "supplied to",
-		"errorThird": "It is required if you show back button"
+		"errorFirst": "no válido",
+		"errorSecond": "suministrada a",
+		"errorThird": "Es necesario si se muestra el botón Atrás"
 	},
 	"setupOverview": {
-		"placeholder": "Enter a name"
+		"placeholder": "Ingresa un nombre"
```
Comment (davidsingal) [inline app/locales/es.json]: Introduzca un nombre

File: app/locales/es.json [hunk 9]
```diff
@@ -1,87 +1,87 @@
...
 	},
 	"setupDrawAreas": {
-		"tapInstruction": "Tap on map to draw"
+		"tapInstruction": "Toque en el mapa para dibujar"
 	},
 	"setupHeader": {
-		"errorFirst": "Invalid prop",
-		"errorSecond": "supplied to",
-		"errorThird": "It is required if you show back button"
+		"errorFirst": "no válido",
```
Comment (davidsingal) [inline app/locales/es.json]: Propiedad no válida
Propiedad inválida
Propiedad incorrecta
Incorrecto

File: app/locales/fr.json [hunk 10]
```diff
@@ -1,87 +1,87 @@
...
-		"loggedIn": "You’re are logged in as",
-		"yourAreas": "Your areas",
-		"logOut": "LOG OUT",
-		"addArea": "ADD AREA",
-		"aboutApp": "About this app",
-		"aboutPartners": "Partners",
-		"aboutTerms": "Terms and Conditions"
+    "title": "Paramètres",
+		"loggedIn": "Vous êtes connecté en tant que",
+		"yourAreas": "Vos domaines",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: As a consistency sake, I would use "zone" again: Vos zones

File: app/locales/fr.json [hunk 11]
```diff
@@ -1,87 +1,87 @@
...
+		"logOut": "SE DÉCONNECTER",
+		"addArea": "AJOUTER ZONE",
+		"aboutApp": "À propos de cette application",
+		"aboutPartners": "Les partenaires",
+		"aboutTerms": "Termes et conditions"
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Conditions d'utilisation

File: app/locales/es.json [hunk 12]
```diff
@@ -1,87 +1,87 @@
...
-		"alert": "alert"
+		"title:": "Alertas",
+		"seeAll": "Ver todo",
+		"alert": "alerta"
 	},
 	"boundaries": {
-		"chooseBoundaries": "Now, choose the boundaries",
-		"selectProtectedArea": "Select a protected area",
-		"drawArea": "Draw an area"
+		"chooseBoundaries": "Ahora, elige los límites",
```
Comment (davidsingal) [inline app/locales/es.json]: Ahora, elija los límites

File: app/locales/fr.json [hunk 13]
```diff
@@ -1,87 +1,87 @@
...
+    "takePicture": "Prendre une photo",
+    "datePlaceholder": "Sélectionner une date",
+    "saveLaterTitle": "Garder pour plus tard",
+    "saveLaterDescription": "Il apparaîtra comme un brouillon dans la section Rapports",
+    "inputText": "Veuillez écrire quelque chose",
+    "inputRadio": "Choisissez une option pour continuer",
+    "inputSelect": "Choisissez au moins une option pour continuer",
+    "input": "S'il vous plaît, répondez à la question",
+    "completed": "Terminé",
+    "uploadAll": "Télécharger tout",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Tout sauvegardé

It means "save" but we don't have a way to say upload correctly.

File: app/locales/fr.json [hunk 14]
```diff
@@ -1,87 +1,87 @@
...
+    "drafts": "Brouillons",
+    "uploaded": "Envoyé"
```
Comment (clementprdhomme) [inline app/locales/fr.json]: "Sauvegardé" is a better I think

File: app/locales/es.json [hunk 15]
```diff
@@ -1,87 +1,87 @@
...
-		"aboutApp": "About this app",
-		"aboutPartners": "Partners",
-		"aboutTerms": "Terms and Conditions"
+    "title": "Ajustes",
+		"loggedIn": "Has iniciado sesión como",
+		"yourAreas": "Tus áreas",
+		"logOut": "Cerrar sesión",
+		"addArea": "AÑADIR ÁREA",
+		"aboutApp": "Acerca de esta aplicación",
+		"aboutPartners": "Socios",
```
Comment (davidsingal) [inline app/locales/es.json]: Colaboradores

File: app/locales/fr.json [hunk 16]
```diff
@@ -1,87 +1,87 @@
...
-		"alerts": "See my alerts",
-		"reports": "Make a report",
-		"myReports": "My reports",
-		"uploadData": "Saved and uploaded data",
-		"map": "Map"
+		"title": "Panel",
+		"alerts": "Voir mes alertes",
+		"reports": "Faire un rapport",
+		"myReports": "Mes rapports",
+		"uploadData": "Données sauvegardées et téléchargées",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Données sauvegardées will do both. French doesn't have a word to say "upload". "Saved" is sufficient I think.

File: app/locales/es.json [hunk 17]
```diff
@@ -1,87 +1,87 @@
...
-		"title": "Dashboard",
+		"title": "Cuadro de mandos",
 		"alerts": "Ver mis alertas",
-		"reports": "Make a report",
-		"myReports": "My reports",
-		"uploadData": "Saved and uploaded data",
-		"map": "Map"
+		"reports": "Hacer un informe",
+		"myReports": "Mis informes",
+		"uploadData": "Datos guardados y cargados",
```
Comment (davidsingal) [inline app/locales/es.json]: Datos guardados y subidos
Datos subidos

File: app/locales/fr.json [hunk 18]
```diff
@@ -1,87 +1,87 @@
...
 	},
 	"setupDrawAreas": {
-		"tapInstruction": "Tap on map to draw"
+		"tapInstruction": "Appuyez sur la carte pour dessiner"
 	},
 	"setupHeader": {
-		"errorFirst": "Invalid prop",
-		"errorSecond": "supplied to",
-		"errorThird": "It is required if you show back button"
+		"errorFirst": "Prop non valide",
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Propriété invalide

I don't know the context but my translation is far better.

File: app/locales/fr.json [hunk 19]
```diff
@@ -1,87 +1,87 @@
...
+		"errorSecond": "fourni à",
+		"errorThird": "Il est nécessaire si vous affichez le bouton"
```
Comment (clementprdhomme) [inline app/locales/fr.json]: Requis pour afficher le button de retour

I don't understand the English sentence but the current French version sounds weird.

**Comment by j8seangel** [comment]
Thanks @vizzcrew !

**Pull request merged**